cmis_web_proxy/static/src/cmis_folder/cmis_folder.js

```diff
@@ -0,0 +1,52 @@
+/** @odoo-module **/
+
+/* ---------------------------------------------------------
++ * Odoo cmis_web
++ * Authors Laurent Mignon 2016, Maxime Franco 2023 Acsone SA/NV
++ * License in __openerp__.py at root level of the module
++ *---------------------------------------------------------
++*/
+
+import {CmisFolderField} from "@cmis_web/cmis_folder/cmis_folder";
+import {patch} from "@web/core/utils/patch";
+
+patch(CmisFolderField.prototype, "open_with_proxy", {
+    getCmisObjectWrapperParams() {
+        const params = this._super(...arguments);
+        params.applyOdooSecurity = this.backend.apply_odoo_security;
+        return params;
+    },
+
+    genCmisSessionToken() {
+        return JSON.stringify({
+            model: this.props.record.resModel,
+            res_id: this.props.record.resId,
+            field_name: this.props.name,
+        });
+    },
+
+    setCmisSessionToken() {
+        if (this.backend.apply_odoo_security) {
+            this.cmisSession.setToken(this.genCmisSessionToken());
+        }
+    },
+
+    async setRootFolderId() {
+        var self = this;
+        self.setCmisSessionToken();
+        this._super(...arguments);
+    },
+
+    getPreviewUrlParams() {
+        // Pas sur de l'utilité de la méthode je n'ia trouvé aucun appel à cette fonction
```

Comment:
I am quite sure you can remove it. Could we try without ?

Reply:
Actually, reading https://github.com/acsone/alfodoo/pull/184, I think we need the token to be set.

I guess this code is part of the mechanism to set the token, but it should be integrated somehow.

Reply:
I read the code since the V10, there is never a call to this method. The token is generated during the generation of the cmis folder field (inside the set_root_folder_id function, that is now called setRootFolderId).
I think it's just a leftover from a previous version to v10. 
I suggest to remove that function to get a clear V16 module.